src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/QueryIT.java

```diff
@@ -784,6 +785,62 @@ public void orderByAscFieldWithSpaceTest() throws IOException {
         Assert.assertEquals("The list is not in ascending order", sortedTestFields, testFields);
     }
 
+    @Test
+    public void testWhereWithBoolCondition() throws IOException {
+        JSONObject response = executeQuery(
+                    StringUtils.format("SELECT * " +
+                                    "FROM %s/account " +
+                                    "WHERE male = true ORDER BY age LIMIT 5",
```

Comment:
Could you please add tests with `false` value as well? Ideally we would have True/true/TRUE/False/false/FALSE, to make sure it works fine with different casing.